### DISCUSSION TITLE
How should we document the purpose of files?

### DISCUSSION THREAD

**Discussion by jmarshvaile** [40dm/gam#2] [Q&A] [answered]
@isaachulvey 

Example: Issue #1 

How should we go about documenting the purpose of files like these? 
- in the file itself
- in a central location (such as a/the readme)
- both
- other

I lean towards in the file as a comment at the top of the file. Seems easier to remember to do it, and easier to maintain.

I think a standard format would be good too. Something that says "I am describing to you what the purpose of this file is"

In the linked example, there is a bullet point list that I believe describes the purpose but that wasn't clear to me until after I read through the file.

The questions:
1. Where should we document what a file does
2. What should be the formatting of that documentation

**Comment by isaachulvey** [comment, marked as answer]
I would suggest in the file as well.

**Reply by jmarshvaile** [reply]
k

**Comment by isaachulvey** [comment]
Ok cool
